code/game/objects/items/weapons/grenades/chem_grenade.dm

```diff
@@ -557,3 +557,26 @@
 /obj/item/weapon/grenade/chem_grenade/timer/New()
 	..()
 	detonator = new/obj/item/device/assembly_holder/timer_igniter(src)
+
+/obj/item/weapon/grenade/chem_grenade/teargas
+	name = "teargas grenade"
+	desc = "Used for training and crowd control operations. Contents under pressure. Do not directly inhale contents."
+	stage = GRENADE_STAGE_COMPLETE
+	path = PATH_STAGE_CONTAINER_INSERTED
+	
+/obj/item/weapon/grenade/chem_grenade/teargas/New()
+	..()
+	var/obj/item/weapon/reagent_containers/glass/beaker/B1 = new(src)
+	var/obj/item/weapon/reagent_containers/glass/beaker/B2 = new(src)
```

Comment:
Shouldn't these be /obj/item/weapon/reagent_containers/glass/beaker/large? Small beakers only hold 50u.